src/mango/test/03-operator-test.py

```diff
@@ -75,6 +75,21 @@ def test_keymap_match(self):
         docs = self.db.find({"foo": {"$keyMapMatch": {"$eq": "aa"}}})
         self.assertEqual(len(docs), 1)
 
+    def test_size(self):
```

Comment:
Could you add another test that shows it handles the case where the `sizeGte` value is not an integer and another when it is acting on a field that is not an array.

Reply:
I have added tests for failures but I don't know how to "assert failure" in this test script (need help).